### PR TITLE
Stop/Start/Restart and Reset commands from CLI

### PR DESCRIPTION
**Pull request by aaronthorp**
These are tasks I like to be able to do with my projects, just added them to the command line as well as adding the secret `reconfig` command that was missing from the help list. These might be handy for others as I have heard in the #meteor IRC channel as well as some issues posted too, so pull request for your consideration :)

Commands that I have made available:

```
mup start
mup stop
mup restart
mup reset
```

Please test this on sunos, I can only assume that all the commands will work, `start`/`stop`/`restart` definitely will but need to confirm the `reset`.

### CODE REVIEW

**Comment by arunoda** [comment]
Thanks this is great. Don't worry about sunos. I'm going to remove it anyway.

**Comment by arunoda** [comment]
I think, we need to think a bit about the reset. It's okay to remove the db. But, since most of the production apps uses some external DB, this command became something tricky. So, shall I purpose something like this.
- It will delete the DB if exists
- It will stop the current deployment 
- Remove all the files related to that (in the /opt directory)
- And remove the /etc/init/<appName>.conf as well

File: lib/taskLists/linux.js [hunk 1]
```diff
@@ -109,6 +109,55 @@ exports.reconfig = function(env, appName) {
   return taskList;
 };
 
+exports.restart = function(env, appName) {
```
Comment (arunoda) [inline lib/taskLists/linux.js]: You don't need to app `env` here. This applies to all the functions in here.

**Comment by aaronthorp** [comment]
haha yeah I was just thinking that as I was going back through it, maybe reset only works if mongo ins installed locally? so if you are running a local instance then it will reset the same as a meteor app, if you are running external database it just fails with a warning?

File: lib/actions.js [hunk 2]
```diff
@@ -142,6 +142,58 @@ Actions.prototype.reconfig = function() {
...
+    sessionsInfo.sessions.forEach(function(session) {
+      var env = _.extend({}, self.config.env, session._serverConfig.env);
+      var taskList = sessionsInfo.taskListsBuilder.reset(
+        env, self.config.appName);
+      taskList.run(session);
+    });
+  }
+};
+
+Actions.prototype.restart = function() {
```
Comment (arunoda) [inline lib/actions.js:145]: You can write this like follows:

``` js
Actions.prototype.reset = function() {
  var self = this;
  for(var os in self.sessionsMap) {
    var sessionsInfo = self.sessionsMap[os];
    var taskList = sessionsInfo.taskListsBuilder.reset(self.config.appName);
    taskList.run(sessionsInfo.sessions);
  }
};
```

You can do same for others as well.

**Comment by arunoda** [comment]
Anyway, I feel some guilty about implementing `mup reset`. Shall we remove it. 
Let's think about that later.

**Comment by aaronthorp** [comment]
yeah, it could be dangerous, thats why i made it seperate commits, might do to leave it out then until more clear on a procedure for it. I will remove and have made above changes also and add to pull?

**Comment by arunoda** [comment]
Yes. Do the changes as I commented to this PR.

**Comment by aaronthorp** [comment]
there we go

**Comment by arunoda** [comment]
Awesome.  Let me do a test my self.

**Comment by arunoda** [comment]
Published as `0.9.0`

**Pull request merged**